package.json

```diff
@@ -189,5 +195,24 @@
       "jest",
       "babel-jest"
     ]
+  },
+  "detox": {
+    "test-runner": "jest",
+    "runner-config": "e2e/native/config.json",
+    "specs": "e2e",
+    "configurations": {
+      "ios.sim.debug": {
+        "binaryPath": "native/ios/build/Build/Products/Debug-iphonesimulator/native.app",
+        "build": "xcodebuild -project native/ios/native.xcodeproj -scheme native -configuration Debug -sdk iphonesimulator -derivedDataPath native/ios/build",
```

Comment:
This works? Strange if so because it doesn't include the pods. Should be referencing the workspace not the project

Reply:
we can reference the workspace with the --workspace flag, but it works, given we have to install gems and pod to have `npm run ios` working I think we can safely leave it like this

Reply:
I'd be more comfortable if we used the workspace reference I feel like it working is a fluke

Reply:
It did work on Matt's machine freshly updated to high sierra and no detox installed at all, it's super easy to update tough so I'll update it 😄

Reply:
Done 👍